gini/gini-switch-sdk-ios
# Remove motion manager

# Introduction

The motion manager was used to change the image orientation as the device tilts. This is not needed because the image needs to stay in portrait even if the device is rotated to another orientation

# How to test

* Go to the camera screen
* Tilt the device in landscape or up side down orientation
* Take a picture
* In the review screen make sure the image is not rotated

# Merge info

Automatic
